jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/util/TestsUtil.java

```diff
@@ -53,14 +52,8 @@
             logger.warn("Folder with DDLs doesn't exist {}", folderWithDDLs);
             return new File[0];
         }
-        
-        Predicate<File> filterExtension = file -> file.getName().toLowerCase().endsWith(".sql");
-        Predicate<File> filterSpringboot = file -> file.getName().toLowerCase().contains("springboot");
-        Predicate<File> filterName = file -> file.getName().contains("drop");
-        Predicate<File> filter = filterExtension.and((!scriptFilter.isCreate()) ? filterName : filterName.negate())
-                                                .and((scriptFilter.isSpringboot()) ? filterSpringboot : filterSpringboot.negate());
-        
-        File[] foundFiles = Arrays.asList(folderWithScripts.listFiles()).stream().filter(filter).toArray(File[]::new);
+
+        File[] foundFiles = Arrays.asList(folderWithScripts.listFiles()).stream().filter(scriptFilter.build()).toArray(File[]::new);
```

Comment:
`folderWithScripts.listFiles()` might be null

```suggestion
        File[] foundFiles = Arrays.stream(Objects.requireNonNull(folderWithScripts.listFiles())).filter(scriptFilter.build()).toArray(File[]::new);
```